### PR TITLE
springboot v2.7.3

### PR DESCRIPTION
**Pull request by AbdelHedhili**
Signed-off-by: Abdelsalem <abdelsalem.hedhili@rte-france.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
     <groupId>org.gridsuite</groupId>
     <artifactId>gridsuite-dependencies</artifactId>
-    <version>19-SNAPSHOT</version>
```
Comment (geofjamg) [inline pom.xml:24]: Keep 19-SNAPSHOT, 19 will be for next release in another PR

**Review by laurgarn — APPROVED** [review]
Nothing questionable found.
